### ISSUE TITLE
Breaks the "Add a line item" option in the CP order/cart edit

### ISSUE THREAD

**Issue by joshangell** [verbb/Events#63]
**Description**
When you edit a cart or order in the CP having Events installed breaks the "Add a line item" option when editing an order or cart.

**Steps to reproduce**
1. Create an order in the CP or make one on the front-end.
2. Click "Add a line item" and search for anything.
3. In the browser console the XHR request to `/admin/actions/commerce/orders/purchasables-table?sort=&page=1&per_page=10&search=book` returns a 500 error with the response:

```json
{
    "error":"Getting unknown property: verbb\\events\\elements\\Ticket::priceAsCurrency"
}
```

**Additional info**
- Plugin version: 1.4.12
- Craft version: 3.5.2 (Pro)
- Commerce version: 3.2.1 (Pro)
- Multi-site: Yes

**Comment by engram-design** [comment]
Yeah, looks like a Commerce 3.5.x change, but I can't actually see where `priceAsCurrency ` is documented. Looking into it, as it effects a few other custom purchasables.

**Comment by joshangell** [comment]
Ace, I figured it was.

**Comment by engram-design** [comment]
@joshangell Refer to Commerce's issue - https://github.com/craftcms/commerce/issues/1646

**Comment by engram-design** [comment]
Fixed in [Commerce 3.2.2](https://github.com/craftcms/commerce/releases/tag/3.2.2)
